modules/apps/collaboration/wiki/wiki-web/src/main/resources/META-INF/resources/wiki/view_page_details.jsp

```diff
@@ -27,8 +27,7 @@ if (wikiPage != null) {
 	attachmentsFileEntries = wikiPage.getAttachmentsFileEntries();
 }
 
-int numOfVersions = WikiPageLocalServiceUtil.getPagesCount(wikiPage.getNodeId(), wikiPage.getTitle());
-WikiPage initialPage = (WikiPage)WikiPageLocalServiceUtil.getPages(wikiPage.getNodeId(), wikiPage.getTitle(), numOfVersions - 1, numOfVersions).get(0);
+WikiPage initialPage = (WikiPage)WikiPageLocalServiceUtil.getPages(wikiPage.getNodeId(), wikiPage.getTitle(), 0, 1).get(0);
```

Comment:
Got it

I'm going to made a litle change and resend it to you, could you recheck my changes?

Thanks!